### PR TITLE
Allow for a longer title banner if the map name is too long

### PR DESCRIPTION
**Pull request by maddie480**
This allows for longer map names:

image

This is currently part of Collab Utils, but is only applied to collabs (that have to depend on Collab Utils) to avoid confusion ("the title sticks out on my playtesters' PC because they don't have Collab Utils installed except I don't even know that behavior comes from Collab Utils to start with"), and non-collab maps could benefit from this.

### CODE REVIEW

File: Celeste.Mod.mm/Celeste.Mod.mm.csproj [hunk 1]
```diff
@@ -86,6 +86,7 @@
     <Content Include="..\lib-mono\**\*">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <None Remove="Content\Graphics\Atlases\Gui\areaselect\title.png" />
```
Comment (coloursofnoise) [inline Celeste.Mod.mm/Celeste.Mod.mm.csproj]: Is this line needed?

File: Celeste.Mod.mm/MonoModRules.cs [hunk 2]
```diff
@@ -1960,6 +1961,15 @@ bgt.un.s	ldloca.s
...
                 throw new Exception("Incorrect number of matches for string starting with \"areaselect/card\".");
+
+            cursor.Index = 0;
+
+            // Resize the title if it does not fit.
+            while (cursor.TryGotoNext(instr => instr.MatchLdcR4(-60))) {
+                cursor.Emit(OpCodes.Ldarg_0);
+                cursor.Index++;
+                cursor.Emit(OpCodes.Call, m_FixTitleLength);
+            }
```
Comment (coloursofnoise) [inline Celeste.Mod.mm/MonoModRules.cs]: Everest patches should fail hard wherever possible, similar to the code above this.

**Review by coloursofnoise — CHANGES_REQUESTED** [review]

**Comment by maddie480** [comment]
I didn't test the changes I just made from mobile, will test when I can 🤔

**Comment by maddie480** [comment]
it works :+1:

**Review by coloursofnoise — APPROVED** [review]

**Pull request merged**